### PR TITLE
[PDR-11593][feat(reader)]logkit tailx 支持同时采集多个路径模式串

### PR DESCRIPTION
**Pull request by PapaPiya**
## Changes
   
- [ ] feature1
- [ ] feature2
- [ ] fixbug1
- [ ] fixbug2
 
## Reviewers

- [ ] @[someone] please review
- [ ] @[someotherone] please review

## Wiki Changes
 
- options1...
- options2...
    
## Checklist
   
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] Wiki updated

### CODE REVIEW

File: reader/tailx/tailx.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package tailx
 import (
 	"errors"
 	"fmt"
+	"github.com/bmatcuk/doublestar"
```
Comment (redHJ) [inline reader/tailx/tailx.go:6]: 顺序

**Comment by redHJ** [comment]
检查一下其他地方的glob，看看是不是都替换掉比较好

**Comment by PapaPiya** [comment]
> 检查一下其他地方的glob，看看是不是都替换掉比较好

done， 改了 ftp reader 日志获取，配置文件跟clean log部分的filepath.Glob没改

File: reader/tailx/tailx.go (hunk 1 above)
Comment (PapaPiya) [reply reader/tailx/tailx.go]: > 顺序

done

**Comment by redHJ** [comment]
lgtm

**Pull request merged**